reverse/reverse.py

```diff
@@ -38,10 +38,14 @@ def contains(self, value):
 
         return False
 
-    def reverse_list(self, node, prev):
+    def reverse_list(self, node, prev):        
         if self.value = None:
```

Comment:
You're on the right track here. Very close! Think about the case where the linked list is empty, can we access the attribute .value if the node itself is non-existence?